app.py

```diff
@@ -119,12 +119,33 @@ def auth_granted():
     @app.route('/api/slack/commands', methods=['POST'])
     def slash_command():
         data = request.form.to_dict()
+        print('data: ', data)
 
         if data.get('type'):
-            return(get_challenge_response(data, app.config['SLACK_VERIFICATION_TOKEN'])) 
+            return(get_challenge_response(data, app.config['SLACK_VERIFICATION_TOKEN']))
         else:
-            return(jsonify({ "text": "Test reply: " + data['text'] + " from " + data['user_name'] }))
-
+            # /api/slack/commands with no params - give top 5 received emojis for current user
+            if(data.get('text') == ''):
+                slack_user_id = data.get('user_id')
+                user = User.query.filter_by(slack_id=slack_user_id).first()
+                if user:
+                    reaction_counts = {}
+                    for reaction in user.reactions_received:
+                        if reaction.name not in reaction_counts:
+                            reaction_counts[reaction.name] = 1
+                        else:
```

Comment:
i like to use this type of pattern for counting in dictionaries, instead of if-else

`reaction_counts[reaction.name] = reaction_counts.get(reaction.name, 0) + 1`